### DISCUSSION TITLE
Dynamic SSO

### DISCUSSION THREAD

**Discussion by anthonycorbacho** [aarondl/authboss#344] [Q&A]
Hello, we are considering using authboss but our use case is a bit different, our user can have different type of sso based on a organization (one account can have multiple org access). Each org can decide to enable SSO.

Basically we would like to know if auth boss can dynamically (at run time) handle different sso config?

**Comment by aarondl** [comment]
It's definitely not the intended use case. I would try developing your own module. There's likely a way within the existing framework that you could create such an authentication module with authboss.
